src/__tests__/formatPkg.test.ts

```diff
@@ -155,11 +155,11 @@ describe('adds vue-cli plugins', () => {
     name: '@dogs/vue-cli-plugin-dogs',
   };
 
-  const formattedDogs = formatPkg(pkg);
```

Comment:
used at line 182, is that fixed in a different PR?

Reply:
Line 182 is a different describe, or maybe I'm miss reading

Reply:
it's a different describe, but it uses the same variables, right?

Reply:
I'm not sure I get the issue. I'v moved the 3 lines to not be in a describe but in the `test` so that it's not executed every time when you filter some test

Reply:
my bad, I totally misread it, thought it was in a beforeEach or something first

Reply:
ah, no worries ☺️